components/CreateCollectiveMiniForm.js

```diff
@@ -299,7 +300,13 @@ const CreateCollectiveMiniForm = ({
                   value={values.website}
                 >
                   {inputProps => (
-                    <Field as={StyledInput} {...inputProps} placeholder="i.e. opencollective.com" width="100%" />
+                    <Field
+                      as={StyledInput}
+                      {...inputProps}
+                      placeholder="i.e. opencollective.com"
+                      width="100%"
+                      data-cy="mini-form-website-field"
```

Comment:
It's usually ok to target inputs with `input[name="website"]`, it's a good way to test accessibility at the same time (by making sure there is an actual input, and that its name is properly set). But no change requested here, using `data-cy` is the default best practice!